xmppserver/src/main/webapp/group-edit.jsp

```diff
@@ -570,12 +569,12 @@
             <input type="hidden" name="addMember" value="addMember"/>
             ${listPager.hiddenFields}
 
-            <table cellpadding="3" cellspacing="1" border="0" style="margin: 0 0 8px 0;">
+            <table style="margin: 0 0 8px 0;">
                 <tr>
-                    <td nowrap width="1%">
+                    <td style="width: 1%; white-space: nowrap">
                         <fmt:message key="group.edit.add_user" />
                     </td>
-                    <td nowrap class="c1" align="left">
+                    <td class="c1" style="text-align: left; white-space: nowrap">
                          <input type="text" size="45" name="username"/>
                         &nbsp;<input type="submit" name="addbutton" value="<fmt:message key="global.add" />">
                     </td>
```

Comment:
The add member text input is selected by default when entering the group edit page.
Is this intentional behaviour?
Oh, it's pre-existing.

Reply:
I think it _is_ intentional, as that's the page that is shown directly after a group is created. It stands to reason that after you create a group, you want to add members to it.